pyro/generic/dispatch.py

```diff
@@ -54,37 +53,43 @@ def pyro_backend(*aliases, **new_backends):
 
 _ALIASES = {
     'pyro': {
-        'pyro': 'pyro',
         'distributions': 'pyro.distributions',
+        'handlers': 'pyro.poutine',
         'infer': 'pyro.infer',
-        'optim': 'pyro.optim',
         'ops': 'torch',
+        'optim': 'pyro.optim',
+        'pyro': 'pyro',
     },
     'minipyro': {
-        'pyro': 'pyro.contrib.minipyro',
+        'distributions': 'pyro.distributions',
+        'handlers': 'pyro.poutine',
         'infer': 'pyro.contrib.minipyro',
-        'optim': 'pyro.contrib.minipyro',
         'ops': 'torch',
+        'optim': 'pyro.contrib.minipyro',
+        'pyro': 'pyro.contrib.minipyro',
     },
     'funsor': {
-        'pyro': 'funsor.minipyro',
-        'infer': 'funsor.minipyro',
-        'optim': 'funsor.minipyro',
         'distributions': 'funsor.distributions',
+        'handlers': 'pyro.poutine',
```

Comment:
this should be
```
'handlers': 'funsor.minipyro',
```

Reply:
Done. I forgot that the simplified effect handling abstraction in minipyro is one of the reasons why we have it.